src/components/posts/PostItem.js

```diff
@@ -1,23 +1,24 @@
 import { useState } from "react/cjs/react.development";
 import Post from "../UI/Post";
 import Backdrop from "../UI/Backdrop";
-import DeleteButton from "./buttons/DeleteButton";
 import EditModal from "./buttons/EditModal";
-import LikeButton from "./buttons/LikeButton";
 import classes from "./PostItem.module.css";
+import ButtonAction from "./buttons/ButtonAction";
+import { DeletePost, LikeHandler } from "./buttons/LikeDeleteFunctions";
 
 function PostItem({ likes, content, avatar, id, author, date }) {
   const [showEdit, setShowEdit] = useState(false);
   const [likeCount, setLikeCount] = useState(likes);
   const [contentText, setContentText] = useState(content);
   const [postDelete, setPostDelete] = useState(true);
+  const [showDeleteAlert, setShowDeleteAlert] = useState(false);
 
   function editHandler() {
-    if (showEdit) {
-      setShowEdit(false);
-    } else {
-      setShowEdit(true);
-    }
+    setShowEdit(!showEdit);
```

Comment:
Nice, condenses a lot of code and makes things a lot clearer

Reply:
thanks